libs/ledger-live-common/src/config/index.ts

```diff
@@ -1,11 +1,20 @@
 import { CryptoCurrency, CryptoCurrencyId } from "@ledgerhq/types-cryptoassets";
 import { ConfigInfo, LiveConfig } from "@ledgerhq/live-config/LiveConfig";
-import { CurrencyConfig } from "@ledgerhq/coin-framework/config";
+import { CurrencyConfig, SharedConfig } from "@ledgerhq/coin-framework/config";
 
 export type CurrencyLiveConfigDefinition = Partial<
   Record<`config_currency_${CryptoCurrencyId}`, ConfigInfo>
 >;
 
+const getSharedConfiguration = (): SharedConfig => {
+  const config = LiveConfig.getValueByKey(`config_currency`);
+  if (!config) {
+    throw new Error(`Configuration config_currency not found, please check Firebase Remote Config`);
+  }
+
+  return config as SharedConfig;
```

Comment:
This is `any` anyway
```suggestion
  return config;
```

Reply:
done